Automattic/jetpack
# CLI: Cleaner output for cli subcommands.

When running `jetpack cli link` (directly or via `yarn cli-link`), there is a warning outputted on the second and following executions indicating that the CLI is already globally available.

This is just a warning, but can lead to confusion when there are issues.

#### Changes proposed in this Pull Request:

* The default experience will no longer show the direct output, but only a success message. On error, it'll be verbose and include it.
* Can pass `-v` either in direct `jetpack cli link` or `yarn cli-link` execusion to show verbose output, including the warning.

#### Jetpack product discussion


n/a

#### Does this pull request change what data or activity we track or use?


No.

#### Testing instructions:






* Run `yarn cli-link` and `yarn cli-link -v` a few times.
* Run `yarn cli-unlink` and `yarn cli-unline -v`. Note: If you run `yarn cli-unlink` when the CLI isn't linked, it will error. That's expected at this time. To test unlink multiple times, run a `yarn cli-link` variant above between `yarn cli-unlink` executions.
